sympy/physics/mechanics/tests/test_jointsmethod.py

```diff
@@ -143,3 +147,62 @@ def test_chaos_pendulum():
     assert (rhs[0] - xd).simplify() == 0
     xd = (IBxx - IByy) * omega**2 * sin(phi) * cos(phi) / IBzz
     assert (rhs[1] - xd).simplify() == 0
+
+def test_four_bar_linkage_with_manual_constraints():
```

Comment:
This is a numerical test I've created, apparently I cannot import ``array`` from ``numpy``. But do you also agree with this setup to test systems, where the equations of motion become bigger?

Reply:
Oh yeah, since numpy is an optional dependency of sympy, you would have to accommodate that.